feat(content) Northern Anarchist Map Changes (#11604)
endless-sky/endless-sky: data/map planets.txt

		threshold 5500
		fleet "Large Republic" 6

planet "Kennecott"
	attributes frontier north
	landscape land/badlands9-harro
	description `Kennecott can best be described as a world well past its prime. The planet was originally colonized for its lackluster but easily accessible natural resources, which where a requirement for the compact, primitive mining machines available at the dawn of the hyperdrive. For a time, Kennecott was a booming center for industrial goods and technological developments. However, the miniaturization and cost reduction of hyperdrive technology soon saw Kennecott's offerings dwarfed by higher quality imported metals.`
	description `	While a few businesses remain, mostly providing basic goods and tools, all major industries have long since pulled out of Kennecott. Most locals argue that the Syndicate was responsible for this decline, providing cheaper metals to customers like Betelgeuse at a loss to starve out native competition. The remaining population resents the megacorperation, now chained to this dying world by poverty and closed doors.

> 	spaceport `The atmosphere of the last remaining space port can best be described as a dead mall. The wide hallways and extensive landing pads carry a lonely aura, once designed to service hundreds of merchant ships at a time. Dozens of aging stores and facilities line the promenade, most now locked behind heavy metal shutters, cobwebs and rust inching up the desolate metal bars. Still, a few local business still operate, their warm store lights overpowering the dim, tinged sun from dirty skylights above.`

Author: It might be, but I figured it would be the easiest way to convey the kind of _vibe_ the place has without getting to into the nitty-gritty, since I'd imagine most people have a pretty good idea of what a dead mall looks like. If it's too out of place, it's not _crucial_ to the description so we could just axe it if needed!

> 	spaceport `The atmosphere of the last remaining space port can best be described as a dead mall. The wide hallways and extensive landing pads carry a lonely aura, once designed to service hundreds of merchant ships at a time. Dozens of aging stores and facilities line the promenade, most now locked behind heavy metal shutters, cobwebs and rust inching up the desolate metal bars. Still, a few local business still operate, their warm store lights overpowering the dim, tinged sun from dirty skylights above.`

Reviewer: It does feel kind of out of place imo. Dead malls to me feel like a late 20th-early 21st century thing, not a term I'd expect to be relevant 1000 years into the future.

planet Haven
	attributes frontier mining north "north pirate" pirate
	landscape land/dune1
		description `Haven was founded by highly successful "privateers" nearly two centuries ago. Since then, following the fall of The Baroness, it has become the capital for the Northern Anarchist State of Arneb. Numerous small manufacturing shops sell outfits that cannot be bought legally, and there is even a small shipyard where customized ships are sold to those who have the right connections.`
	description `	Most of the population lives in large, urban centers along the few large bodies of fresh water on the planet. The cities themselves are a tightly knit web of roads and towers, surrounded by ancient protective walls and spaceport landing pads. Centuries old high-rises stand shoulder to shoulder with modern skyscrapers, all covered in a mess of dazzling and attention-grabbing holographic signage.`
	description `	Much of the planet is desert, but with enough moisture to grow enough crops to feed a large part of the population. Strip mining has turned other parts of the planet into barren wastelands, and has polluted many of the rivers.`

> 	spaceport `While a few small landing pads lie inches from the city border, a majority of the spaceport is built inside a large trench that wraps around half of the city. Inside this clautstrophobic trench, a flurry of activitiy kicks up loose trash and debris, large moving platforms pulling ships in and out of hangars built along the walls. Dozens of trucks and trams venture deep into tunnels below, a seedy labyrinth of roads and bunkers built long before the city above.`

Reviewer: ```suggestion
	spaceport `While a few small landing pads lie inches from the city border, a majority of the spaceport is built inside a large trench that wraps around half of the city. Inside this claustrophobic structure, a flurry of activity kicks up loose trash and debris, as large moving platforms pull ships in and out of hangars built along the walls. Dozens of trucks and trams venture deep into tunnels below, a seedy labyrinth of roads and bunkers built long before the city above.`
```

	landscape land/loc2
	description `The original founders of Midas Station have long since been forgotten to time. While the citizens of the paradise worlds believe pirate space is nothing more than backstabbing chaos, Midas Station stands in defiance of this idea. Despite not being run by any government or corperation, the orbital refinery has stayed in opperation for decades. The entire workfore is volunteers, taking a small cut of any refined goods to keep the station running.`
	description `	The hallways are tight and claustrophobic, hissing machinery and rumbling droans echo through the dimly lit corridors. Despite the huge, deafening climate control units hanging down above you, the temprature inside is enough to make you break a sweat.`
	spaceport `The spaceport walls are grimy and worn, leaving little room to maneuver a ship inside the cramped hangar. Beneath the grated floor, humming conveyor belts pump a stream of crushed ore towards the station's core, occasionally kicking up small pebbles at passerby. The centeral refinery itself radiates a sinister orange glow, water dripping out of leaky pipes and vaporizing on contact.`

> 	spaceport `	Overlooking it all is a large balcony, bustling with activity. It's here that patient miners await their refined goods, large bricks of questionably pure metals and packaged plates rolled out on conveyor belts akin to a baggage claim, branded with names and weights. Huge overhead gantrys swing by scarily close to the floor, having little space to move, depositing shipments on small self-propelled wagons.`

Reviewer: ```suggestion
	spaceport `	Overlooking it all is a large balcony, bustling with activity. It's here that patient miners await their refined goods: large bricks of questionably pure metals and packaged plates rolled out on conveyor belts akin to a baggage claim, branded with names and weights. Huge overhead gantries swing by scarily close to the floor, having little space to move, depositing shipments on small self-propelled wagons.`
```

	spaceport `Rhyac’s only public space port sits atop the planet’s largest island, nestled around the base of a long dormant volcano. The humid air blows a thin layer of ice across the cold steel landing pads, coating the surface not even an hour after a ship comes in for landing. The few ships owned by the locals here are draped in heavy black tarps that bellow with passing gusts of ice.`
	spaceport `	The main hangar sits nestled inside the shattered remains of an ancient human city ship, catwalks and storefronts bolted along the walls and ceiling of the massive metal structure. A warm orange glow drapes across the room, radiating from heat lamps and low-energy lights. The ramshackle market stalls and bars all huddle around geothermal generators that dot the hangar, spreading a steamy haze across the floors like a fog machine as they melt the condensation around them.`
	"required reputation" -10
   	bribe 0.1
	security 0
	tribute 200
		threshold 2000
		fleet "Large Northern Pirates" 10
		fleet "Small Northern Pirates" 15

planet Dot
	attributes uninhabited
	landscape land/canyon05a

> 	description `While space is limited in the northern pirate kingdom, and Dot is just as inhabitable as any other moon, not a soul has ever set foot on the surface of this dwarf planet with plans on staying. With days as dark as night and a thin atmosphere of near pure nitrogen, the planet has little to offer. Without any strategic importance or valuable materials, Dot is seldom visited by any party.`

Author: Ah, this must be a remnant from the original concept I had for a more cohesive, single entity. I'll have to modify this to fit what we have now!

	outfitter "New Montego"
	"required reputation" -10
	bribe 0.1
	security 0
	tribute 200
		threshold 1
		fleet "Large Northern Pirates" 10
		fleet "Small Northern Pirates" 20

planet Nemo
	attributes uninhabited
	landscape land/station50
	description `The air smells strongly of placeholder.`
	government Uninhabited
	security 0

planet "Caribbean"
	attributes gas "requires: gaslining" uninhabited
	landscape land/occupas
	government Uninhabited
	description `Caribbean, often called "The Gulf of Montego" by the locals, is a calm and temprate gas giant. Monolithic clouds of dense ammonia gas lazily float through the atmosphere, their puffy whites tinted with a heavy teal accent as the sun shines down through the methane-rich atmosphere.`

> 	description `	Back during the caravan era, scientific probes of all sort where dropped into the core of Caribbean. Scans from this time revealed the presence of  seemingly solid massive objects deep within the core, far past the pressure rating of any probe avalible. While little research has been done since, it is believed to be the result of temprature inversions, trapped pockets of cold air that can result in false returns.`

Reviewer: ```suggestion
	description `	Back during the caravan era, scientific probes of all sort were dropped into the core of Caribbean. Scans from this time revealed the presence of seemingly solid massive objects deep within the core, far past the pressure rating of any probe available. While little research has been done since, it is believed to be the result of temperature inversions, trapped pockets of cold air that can result in false returns.`
```

	landscape land/enceladus_1
	description `Although this moon initially seems barren and lifeless, there is evidence that the nearby Gegno Scin have recently attempted to set up an outpost here. Several partially-constructed buildings and landing zones are scattered about the surface, but none of them show any signs of activity, likely due to recent warfare with the Gegno Vi. Despite this, a small number of satellites actively orbit Zydee, likely providing important tactical information such as the volume of system traffic.`

planet "Midas Station"
	attributes frontier north "north pirate" pirate mining
	landscape land/loc2

> 	description `The original founders of Midas Station have long since been forgotten to time. While the citizens of the paradise worlds believe pirate space is nothing more than backstabbing chaos, Midas Station stands in defiance of this idea. Despite not being run by any government or corperation, the orbital refinery has stayed in opperation for decades. The entire workfore is volunteers, taking a small cut of any refined goods to keep the station running.`

Reviewer: ```suggestion
	description `The original founders of Midas Station have long since been forgotten to time. While the citizens of the Paradise Worlds believe pirate space is nothing more than backstabbing chaos, Midas Station stands in defiance of this idea. Despite not being run by any government or corporation, the orbital refinery has stayed in operation for decades. The entire workforce is volunteers, taking a small cut of any refined goods to keep the station running.`
```

	description `	The hallways are tight and claustrophobic, with hissing machinery and rumbling drones echoing through the dimly lit corridors. Despite the huge, deafening climate control units hanging down above you, the temperature inside is enough to make you break a sweat.`

> 	spaceport `The spaceport walls are grimy and worn, leaving little room to maneuver a ship inside the cramped hangar. Beneath the grated floor, humming conveyor belts pump a stream of crushed ore towards the station's core, occasionally kicking up small pebbles at passerby. The centeral refinery itself radiates a sinister orange glow, water dripping out of leaky pipes and vaporizing on contact.`

Reviewer: ```suggestion
	spaceport `The spaceport walls are grimy and worn, leaving little room to maneuver a ship inside the cramped hangar. Beneath the grated floor, humming conveyor belts pump a stream of crushed ore towards the station's core, occasionally kicking up small pebbles at passerby. The central refinery itself radiates a sinister orange glow, water dripping out of leaky pipes and vaporizing on contact.`
```

	spaceport `	Overlooking it all is a large balcony, bustling with activity. It's here that patient miners await their refined goods: large bricks of questionably pure metals and packaged plates rolled out on conveyor belts akin to a baggage claim, branded with names and weights. Huge overhead gantries swing by scarily close to the floor, having little space to move, depositing shipments on small self-propelled wagons.`
	"required reputation" -10
	bribe 0.1
	security 0
	tribute 200
		threshold 2000
		fleet "Large Northern Pirates" 5
		fleet "Small Northern Pirates" 10
		fleet "Priate Miners" 5

planet Rhyac
	attributes frozen fishing "ice giant" frontier north "north pirate" pirate
	landscape land/snow9

> 	description `Rhyac teeters at the end of the habitable zone, sparsely populated by only two kinds of people. The desperate in need of hiding, and those who hunger for a challenge. The planet is almost entirely covered in a frozen ocean, with tiny islands no bigger than a small earth city. Fluffy, dense clouds loom overhead, the few rays of light passing through refracting off small patches of mirror-like ice in a sparkly shimmer.`

Reviewer: ```suggestion
	description `Rhyac teeters at the end of the habitable zone, sparsely populated by only two kinds of people: those desperate for a hiding place, and those who hunger for a challenge. The planet is almost entirely covered in a frozen ocean, with tiny islands no bigger than a small city. Fluffy, dense clouds loom overhead, the few rays of light passing through refracting off small patches of mirror-like ice in a sparkly shimmer.`
```

	description `	Ice fishing is a big business on Rhyac, as most of the land is a jet black volcanic rock, cold and infertile. The locals glide across the frozen ocean on hovercraft, using large harpoon cannons to skewer the gigantic worm-like whales that skirt beneath the ice surface. Most of the inhabitants live in small shacks or hidden bunkers along the equator, with the true population being unknown.`

> 	spaceport `Rhyac’s only public space port sits atop the planet’s largest island, nestled around the base of a long dormant volcano. The humid air blows a thin layer of ice across the cold steel landing pads, coating the surface not even an hour after a ship comes in for landing. The few ships owned by the locals here are draped in heavy black tarps that bellow with passing gusts of ice.`

Reviewer: ```suggestion
	spaceport `Rhyac’s only public spaceport sits atop the planet’s largest island, nestled around the base of a long dormant volcano. The humid air blows a thin layer of ice across the cold steel landing pads, coating the surface not even an hour after a ship comes in for landing. The few ships owned by the locals here are draped in heavy black tarps that bellow with passing gusts of ice.`
```

> 	spaceport `	The main hangar sits nestled inside the shattered remains of an ancient human city ship, catwalks and storefronts bolted along the walls and ceiling of the massive metal structure. A warm orange glow drapes across the room, radiating from heat lamps and low-energy lights. The ramshackle market stalls and bars all huddle around geothermal generators that dot the hangar, spreading a steamy haze across the floors like a fog machine as they melt the condensation around them.`

Reviewer: ```suggestion
	spaceport `	The main hangar sits nestled inside the shattered remains of an ancient city ship, catwalks and storefronts bolted along the walls and ceiling of the massive metal structure. A warm orange glow drapes the room, radiating from heat lamps and low-energy lights. The ramshackle market stalls and bars all huddle around geothermal generators that dot the hangar, spreading a steamy haze across the floors like a fog machine as they melt the condensation around them.`
```

	"required reputation" -10
   	bribe 0.1
	security 0
	tribute 200
		threshold 2000
		fleet "Large Northern Pirates" 10
		fleet "Small Northern Pirates" 15

planet Dot
	attributes uninhabited
	landscape land/canyon05a
	description `While space is limited in the northern pirate kingdom, and Dot is just as inhabitable as any other moon, not a soul has ever set foot on the surface of this dwarf planet with plans on staying. With days as dark as night and a thin atmosphere of near pure nitrogen, the planet has little to offer. Without any strategic importance or valuable materials, Dot is seldom visited by any party.`
	description `	The surface of the planet is unremarkable, save for occasional asteroid impacts or debris from combat overhead. It's believed Dot was flung out of its orbit by a rogue planet long ago, now slowly drifting away from Hatysa until it leaves the system and is cast into eternal darkness between the stars.`
	government Uninhabited
	security 0

planet Masquerade
	attributes farming frontier north "north pirate" pirate
	landscape land/masquerade

> 	description `From orbit, Masquerade looks entirely uninhabbitable. The ozone layer is too thin to block most incoming radiation, leaving the surface nearly dead and incredibly radioactive. However, the planet is home to towering mountains that flank vein-like canyons all across the planet's surface. These sheir cliffs condense Masquerade's humid air into thick, fluffy clouds that hover below the peaks. It's under these clouds that life can thrive, mostly shielded from the harmful gamma radiation above.`

Reviewer: ```suggestion
	description `From orbit, Masquerade looks entirely uninhabitable. The ozone layer is too thin to block most incoming radiation, leaving the surface nearly dead and incredibly radioactive. However, the planet is home to towering mountains that flank vein-like canyons all across the planet's surface. These sheer cliffs condense Masquerade's humid air into thick, fluffy clouds that hover below the peaks. It's under these clouds that life can thrive, mostly shielded from the harmful gamma radiation above.`
```

	spaceport `	Most major settlements are positioned far away from the spaceports themselves, as the frequent passage of starships leaves deadly holes in the cloud cover above. The cities themselves tightly hug the walls of the canyon, burried deep within the thick stone with tunnels that can stretch killometers long. Lofty catwalks and bridges dance between sturdy stone towers while farmers wizz by on well-worn ontricopters, carrying all sorts of humid crops like rices and beans.`
	shipyard "Basic Ships"
	shipyard "Northern Pirates"
	shipyard "True Anarchist"
	outfitter "Ammo North"
	outfitter "Pirate Outfits"
	outfitter "Pirate Basics"
	outfitter "Northern Pirate Basics"
	outfitter "Common Outfits"
	outfitter "Delta V Basics"
	outfitter "True Anarchist"
	"required reputation" -10
	bribe 0.1
	security 0
	tribute 200
		threshold 2000
		fleet "Large Northern Pirates" 5
		fleet "Small Northern Pirates" 10

planet "New Montego"
	attributes frontier north "north pirate" pirate factory
	landscape land/newmontego

> 	description `At a glance, New Montego looks like the perfect tropical resort. Beautiful beaches, bright oceans, and a stable climate. However, the strong strong odour in the air says otherwise. The oceans here are filled with a volatile concoction of sulfur and lithium, perpetually heating the planet's atmosphere to above 310 kelvin year-round, getting their blue tint from the thick slurry of bacteria that seed the seas.`

Reviewer: ```suggestion
	description `At a glance, New Montego looks like the perfect tropical resort: beautiful beaches, bright oceans, and a stable climate. However, the strong strong odor in the air says otherwise. The oceans here are filled with a volatile concoction of sulfur and lithium, perpetually heating the planet's atmosphere to above 310 kelvin year-round, getting their blue tint from the thick slurry of bacteria that seed the seas.`
```

> 	description `	Most land on New Montego is split up into large islands that dot the expansive oceans, beaches accented with bright orange and yellow outcrops that contrast the saturated blue sky. Thick green grass covers most of the land, blanketing islands in a fluffy greenish-yellow carpet. Thick trees punch through the dense foliage, growing outwards instead of upwards, giving their branches a unique mushroom-like look.`

Reviewer: ```suggestion
	description `	Most land on New Montego is split up into large islands that dot the expansive oceans, with beaches accented with bright orange and yellow outcrops that contrast with the saturated blue sky. Thick green grass covers most of the land, blanketing islands in a fluffy greenish-yellow carpet. Thick trees punch through the dense foliage, growing outwards instead of upwards, giving their canopy a unique mushroom-like look.`
```

> 	spaceport `Despite not being safe for swimming, New Montego's oceans have a variety of industrial uses, specifially in the production of batteries. Large industrial pumps leech off the vibrant oceans, excracting and filtering valuable resources from the water through a snaking network of hissing, dripping pipes that weave over and around buildings like vines.`

Reviewer: ```suggestion
	spaceport `Despite not being safe for swimming, New Montego's oceans have a variety of industrial uses, specifially in the production of batteries. Large industrial pumps leech off of the vibrant oceans, excracting and filtering valuable resources from the water through a snaking network of hissing, dripping pipes that weave over and around buildings like vines.`
```

> 	spaceport `	Spaceports on New Montego are more of a collection of loosely bridged landing pads than a single coherent structure, often nested on top of sturdy outcrops and patches of open land between the trees. Small cargo buggies race along the beaches like a highway, kicking up sand that glistens in the sunlight. Sandstone homes and bleached wood huts huddle up against towering metal structures, bridges built atop pipes and dangling from one platform to the next.`

Reviewer: ```suggestion
	spaceport `	Spaceports on New Montego are more of a collection of loosely bridged landing pads than a single coherent structure, often nested on top of sturdy outcrops and patches of open land between the trees. Small cargo buggies race along the beaches like a highway, kicking up sand that glistens in the sunlight. Sandstone homes and bleached wood huts huddle up against towering metal structures. Bridges built atop pipes dangle from one platform to the next.`
```

	outfitter "New Montego"
	"required reputation" -10
	bribe 0.1
	security 0
	tribute 200
		threshold 1
		fleet "Large Northern Pirates" 10
		fleet "Small Northern Pirates" 20

planet Nemo
	attributes uninhabited
	landscape land/station50

> 	description `The air smells strongly of placeholder.`

Reviewer: Needs to be fixed?

	government Uninhabited
	security 0

planet "Caribbean"
	attributes gas "requires: gaslining" uninhabited
	landscape land/occupas
	government Uninhabited
	description `Caribbean, often called "The Gulf of Montego" by the locals, is a calm and temprate gas giant. Monolithic clouds of dense ammonia gas lazily float through the atmosphere, their puffy whites tinted with a heavy teal accent as the sun shines down through the methane-rich atmosphere.`

> 	description `	Back during the caravan era, scientific probes of all sort were dropped into the core of Caribbean. Scans from this time revealed the presence of seemingly solid massive objects deep within the core, far past the pressure rating of any probe available. While little research has been done since, it is believed to be the result of temperature inversions, trapped pockets of cold air that can result in false returns.`

Reviewer: ```suggestion
	description `	Back during the caravan era, scientific probes of all sorts were dropped into the core of Caribbean. Scans from this time revealed the presence of seemingly solid massive objects deep within the core, far past the pressure rating of any probe available. While little research has been done since, it is believed to be the result of temperature inversions, trapped pockets of cold air that can result in false returns.`

```

planet "Anago"
	attributes frontier north "north pirate" pirate mining
	landscape land/mars2alt

> 	description `Anago is the primary source of raw metals for the nearby New Monetgo outfitters and Haven shipyards, the unusually high metal content and low gravity making it the ideal spot for lunar mining camps. It's believed that Anago is the chunk of a formerly large planet, the outer layers stripped clean to reveal the dense, metallic core.`

Reviewer: ```suggestion
	description `Anago is the primary source of raw metals for the nearby New Montego outfitters and Haven shipyards, with the unusually high metal content and low gravity making it the ideal spot for lunar mining camps. It's believed that Anago is what remains of a large terrestrial planet, the outer layers stripped clean to reveal the dense, metallic core.`
```

planet "Dissipation"
	attributes frontier north "north pirate" pirate mining
	landscape land/mercury1
	government Uninhabited
	description `Back during the caravan era, Dissipation was a vital source of helium-3, an important component of early fusion and hyperspace fuels. The dense sand-like surface and very low gravity meant some large mining and tanker vessels could descend to the surface under their own power, which was a rare feat before the invention of modern gravity-repulsor technology.`
	description `	However, following the advent of sophisticated ramscoops and collecting stations, helium-3 from Dissipation slowly decreased in demand. In due time, the dirt-cheap prices of fuels forced any remaining operations out of business.`
	description `	While a few of the empty mines have been transformed into secret hideouts by crime groups and affluent pirates, most industries have long since been abandoned, with dozens of ghost ports now dotting the planet.`

planet "Boneyard"
	attributes uninhabited
	landscape land/mars0
	government Uninhabited

> 	description `Officially, this planet is known simply as Almaaz I. It has poor natural resources, no atmosphere, and is far too close to the sun for any meaningful terraforming. However, a unique combination of a soft, sandy surface and weak gravity leads to Boneyard collecting lots of shipwrecks and debris floating in the Almaaz system. Strong enough to pull wrecks down, but weak enough to not break them apart on impact.`

Reviewer: Is the gravity weak or strong?

> 	description `	Thousands of wrecks litter the surface of this planet, ranging from tiny chunks of daggers and combat drones to (mostly) whole Navy carriers, all picked clean and left to soak in the sun like skeletons in the desert.`

Reviewer: ```suggestion
	description `	Thousands of wrecks litter the surface of this planet, ranging from tiny chunks of Daggers and Combat Drones to (mostly) whole Navy Carriers, all picked clean and left to soak in the sun like skeletons in the desert.`
```

> 	description `	Interestingly, the planet lacks any sort of water for geologic processes like weathering or erosion to create the sandy surface that covers the planet, a fact which has puzzled scientists. It's believed that Boneyard was subject to heavy orbital bombardment at one point, perhaps from rings that once orbited Almaaz II, but there is little evidence to support this theory.`

Reviewer: ```suggestion
	description `	Interestingly, the planet lacks any sort of water necessary for geologic processes like weathering or erosion to create the sandy surface that covers the planet, a fact which has puzzled scientists. It's believed that Boneyard was subject to heavy orbital bombardment at one point, perhaps from rings that once orbited Almaaz II, but there is little evidence to support this theory.`
```

> 	description `	Interestingly, the planet lacks any sort of water for geologic processes like weathering or erosion to create the sandy surface that covers the planet, a fact which has puzzled scientists. It's believed that Boneyard was subject to heavy orbital bombardment at one point, perhaps from rings that once orbited Almaaz II, but there is little evidence to support this theory.`

Reviewer: What's Almaaz II?
